### PR TITLE
build: Replace docz with storybook

### PR DESCRIPTION
**Pull request by darryltec**
## Motivations

It's time to say goodbye to Docz and replace it with Storybook. It still uses the same scripts to run it so no change there.

### Added

- Peer dependencies that `useCollectQuery` hook needs

### Changed

- Moved color swatch component that is under docz-tools into storybook
- Replaced `npm start` and `npm run build` script to run/build storybook instead of docz
- All reference to Docz have been replaced to a Storybook counterpart

### Deprecated

- 

### Removed

- Docz settings and dependencies
- `docz-tools` have been deleted since we don't need it anymore

### Fixed

- 

### Security

- 

## Testing

~Cloudflare build will fail since the settings within it are pointing to a Docz folder that needs to be updated once this branch is released. In the meantime, use this specific build to test based of https://github.com/GetJobber/atlantis/commit/7e1896fff273b57db71fbd4c301cd598c9c7e371~

- ~https://c6021e18.atlantis.pages.dev/~

https://delete-docz.atlantis.pages.dev/?path=/story/introduction--page

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,12 @@
...
     "prestart": "npm run buildHooksIndex",
-    "start": "USE_ATLANTIS_ALIASES=true docz dev",
+    "start": "npm run storybook",
     "start:withPrivateComponents": "USE_ATLANTIS_ALIASES=true PRIVATE_COMPONENTS=visible docz dev",
     "test": "jest",
     "plop": "echo '🚧 `npm run plop` is now `npm run generate`'",
     "generate": "plop -- --path='./packages/components/src/'",
-    "build": "USE_ATLANTIS_ALIASES=true docz build",
-    "serve": "docz serve",
+    "build": "npm run build-storybook -- -o .docz/dist",
```
[the remark this answers is not in the document]
Comment (darryltec) [reply package.json:13]: this is just a test to see if it would deploy. cloudflare is set up to look in to that folder

**Comment by chris-at-jobber** [comment]
Q: does this change the local dev steps at all? Like right now:

1. I pull down a branch 
2. `npm ci`
3. `npm run start` to view Docz docs
4. `npm run storybook` to view Storybook docs

Would `npm run start` now run Storybook?

**Comment by darryltec** [comment]
It would still be the same. Like nothing happened

**Comment by chris-at-jobber** [comment]
@darryltec very nice news! No docs to update 😄

**Comment by darryltec** [comment]
Cloudflare build will fail since the settings within it is pointing to a docz folder that needs to be updated once this branch is released. In the mean time, use this specific build to test based of https://github.com/GetJobber/atlantis/pull/1066/commits/7e1896fff273b57db71fbd4c301cd598c9c7e371

- https://c6021e18.atlantis.pages.dev/

**Comment by chris-at-jobber** [comment]
I'm seeing that every component appears to default to Canvas view: 
image

**Comment by darryltec** [comment]
@chris-at-jobber oh crazy. it's different in local than in cloudflare 🤔

**Comment by darryltec** [comment]
@chris-at-jobber oh wait. now it's working fine forme

**Comment by rodrigoeidelvein** [comment]
@darryltec why did you add `graphql` and `subscriptions-transport-ws`?

**Comment by darryltec** [comment]
@chris-at-jobber oh, it's the ondemand stories. can't use that then haha
@rodrigoeidelvein it's a peer dependency of `@apollo/client` 
image

**Comment by rodrigoeidelvein** [comment]
@darryltec You can delete [this MDX file](https://github.com/GetJobber/atlantis/blob/delete-docz/packages/hooks/README.mdx#L1) too

**Comment by chris-at-jobber** [comment]
Do we have a way to verify that `atlantis.getjobber.com` will in fact send people to the Storybook instance once this is merged?

**Comment by darryltec** [comment]
@chris-at-jobber deploy preview has the same settings for production so if you change 1 and it works, the other would just work as well

@rodrigoeidelvein nope, need to keep that, tho the stories file in the folder is referencing the wrong thing. i'll update it

**Comment by chris-at-jobber** [comment]
Is there a new test build? Assuming the Docs / Canvas issue has been sorted out I'm good with this, but also still seeing the "Canvas by default" behaviour on the build linked in the PR description 👀

**Comment by darryltec** [comment]
@chris-at-jobber  nope. No new build. Would have to put back the workaround I did wait for it to finish and then remove it again and wait for it to finish again 😂. It should just work tho since locally it works fine. And on the existing build, once you visit the page, navigate away, then visit it again, it'll work since it won't have to ondemand build it.

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

No references to Docz and `docz-tools`. Nice to see the `build_docs` steps taking 5 minutes less than before! 🚀 

Bye bye, Docz

**Pull request merged**